### ISSUE TITLE
Allow julia version 1.7.3

### ISSUE THREAD

**Issue by danielcjacobs** [ZeroCM/zcm#392]
Julia has a `dnf` package on Fedora, but it's version 1.7.3, which isn't compatible with the 1.6.0 requirement.

Seems like 1.7.3 still works, so would be nice to allow this version and avoid having to build from source.

**Comment by jbendes** [comment]
@danielcjacobs you can actually just comment out that rule in your local repo if you'd like. If you do that, and verify that zcm compiles and works on that version, we can add it in as a supported version. Really just the 1.6.0 is only because that's the one we've checked out to work properly haha

**Comment by danielcjacobs** [comment]
Yeah I got past the error by modifying the rule. I don't plan on using julia, I just wanted the to build with the `--use-all` flag.

I haven't actually verified that zcm does anything more than compile with that version.
